packages/renderer/src/lib/volume/VolumesList.svelte

```diff
@@ -185,6 +163,48 @@ async function fetchUsageData() {
 function gotoCreateVolume(): void {
   router.goto('/volumes/create');
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+const columns: Column[] = [
+  new Column('Status', 'center', '70px'),
```

Comment:
should we not add the sort comparator on the column object so we have dedicated type there ?

Reply:
```ts
const statusColumn = new Column<ExpectedType>('Status', 'center', '70px');
statusColumn.setSortComparator( and here we can sort using ExpectedType)
```

Reply:
Fixed in latest commit.